integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java

```diff
@@ -87,6 +87,10 @@ public ContainerAllocator(String containerName, int targetNumContainers, int max
     mAllocatedContainers = new ArrayList<>();
   }
 
+  /**
+   * @return a list of all hosts in the cluster which haven't yet reached
+   * the containers per host limit.
```

Comment:
The text should be indented to line up with "a list of ...", see WriteType.java for examples of javadoc line wrapping style

There should be no period at the end